readthedocs-theme/templates/article.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+{% block html_lang %}{{ article.lang }}{% endblock %}
+
+
+{% block title %}{{ article.title|striptags  }} - {{ SITENAME }}{% endblock %}
+
+
+{% block content %}
+
+
+  <!-- Article -->
+  <section class="ui vertical segment padded" id="article">
+    
+    <div class="ui container">
+      <div class="ui grid centered">
+        <div class="column twelve wide computer fourteen wide tablet sixteen wide mobile">
+
+
+          <h1 class="ui header">
+            <span class="ui header">{{ article.title }}</span>
+          </h1>
+
+          <div class="ui basic vertical segment clearing">
+            <div class="ui fitted divider"></div>
+            <div class="ui basic vertical segment left floated">{% for author in article.authors %}{{ author }}{% if not loop.last %}, {% endif %}{% endfor %}</div>
+            <div class="ui basic vertical segment right floated"><time datetime="{{ article.date.isoformat() }}">{{ article.locale_date }}</time></div>
+          </div>
+
+          <img class="ui fluid image" src="{{ article.image or '/theme/img/defaults/alesia-kazantceva-VWcPlbHglYc-unsplash.jpg' }}">
+          {% if article.image and article.image_credit %}
+            <div class="ui basic vertical segment center aligned">
+              <p class="ui"><i>{{ article.image_credit}}</i></p>
+            </div>
+          {% endif %}
```

Comment:
If we decide to show attribution here this might change to something like this

```suggestion
          {% if (article.image and article.image_credit) or not article.image %}
            <div class="ui basic vertical segment center aligned">
              <p class="ui"><i>{{% if not article.image %}Photo by Mailchimp{% else %}{{ article.image_credit}}{% endif %}</i></p>
            </div>
          {% endif %}
```

Reply:
Yeah, we should show attribution with a link to the artist.

EthicalAds shows full attribution and a link:
![image](https://user-images.githubusercontent.com/1140183/154177905-6be10cd5-8dcb-42bf-8419-c245c5ebc8a6.png)

The template logic in your example might be a little off though. It seems it would show an empty `<p>` if there is an `article.image` but no `article.image_credit`.

Reply:
Adding the full attribution. Although this might change if we go for the generative geometric image.

```suggestion
          {% if (article.image and article.image_credit) or not article.image %}
            <div class="ui basic vertical segment center aligned">
              <p class="ui">
                <i>
                  {% if not article.image %}
                    Photo by <a href="https://unsplash.com/photos/VWcPlbHglYc">Mailchimp</a> on <a href="https://unsplash.com/">Unsplash</a>
                  {% else %}
                    {{article.image_credit}}
                  {% endif %}
                </i>
              </p>
            </div>
          {% endif %}
```

For reference, this is how this looks:

![printscreen-16-2-2022](https://user-images.githubusercontent.com/4049894/154231953-75642f4b-b8b2-4749-ba76-6ff5615bb048.jpg)

> The template logic in your example might be a little off though. It seems it would show an empty `<p>` if there is an `article.image` but no `article.image_credit`.

```
{% if (article.image and article.image_credit) or not article.image %}
```
I'm checking if _both_ the article.image `and` article.image_credit are set, in which case I use the article.image_credit bellow, `or` if the article.image does `not` exist, in which case I show the default image credit bellow.

Reply:
Sorry, I was talking about the second conditional:

```{{% if not article.image %}Photo by Mailchimp{% else %}{{ article.image_credit}}{% endif %}```

You're right though that the parent conditional protects this nested conditional. It's just a little confusing due to the similar conditional. I get the logic you're going for now though.